docs/DeveloperGuide.adoc

```diff
@@ -806,9 +806,59 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |sort contacts based on their locality |better match students to tutors
+
+|`* * *` |user |edit the information stored |keep the information up to date
+
+|`* * *` |user |tag tutors and students with multiple values |
```

Comment:
The third part of the user stories is missing. All user stories should be justified with a `so that I can...` part. This is the same for all the other stories with missing information.

Reply:
updated